template/test_example/dev/ansible/00_test_namespace_seperation.yml

```diff
@@ -0,0 +1,15 @@
+- name: Debug all env vars
+  debug:
+    var: ansible_env
+
+- name: some test should not run
+  debug:
+    msg: asd
+  when: lookup( 'env', 'consul_acl' ) | bool and lookup( 'env', 'consul_enterprise' ) | bool
+  tags: test
+
+- name: some test should run
+  debug:
+    msg: asd
+  when: not lookup( 'env', 'nomad_acl' ) | bool
+  tags: test
```

Comment:
```suggestion
```
I wasn't brave enough to click "Delete", but this should be removed, right?

Reply:
Yeah. We can delete it